cmd/virt-handler/virt-handler.go

```diff
@@ -341,6 +341,7 @@ func (app *virtHandlerApp) Run() {
 	launcherClientsManager := launcher_clients.NewLauncherClientsManager(app.VirtShareDir, podIsolationDetector)
```

Comment:
Commit message:
1. > it is not aware of interfaces that are explicitly unplugged for migration, in order to be replugged at the target.

Please consider explicitly referring to SR-IOV NICs, as currently it is unclear if one is unfamiliar with the details.

2. The second link is not permanent and will probably break following the backport of this PR to `release-1.6`.

Same in PR description.